server.js

```diff
@@ -1,19 +1,16 @@
-var deepstream = require("deepstream.io");
```

Comment:
@browniecab can you remove the rest of the changes aside from the two blocks I commented below?